osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs

```diff
@@ -121,7 +121,11 @@ protected override void LoadComplete()
                     // At the point we reach here, it's not guaranteed that all realm writes have taken place (there may be some in-flight).
                     // We are only aware of writes that originated from our own flow, so if we do see one that's active we can avoid handling the feedback value arriving.
                     if (realmWriteTask == null)
+                    {
+                        Current.Disabled = false;
+                        Current.Disabled = allowOffsetAdjust;
```

Comment:
Huh? Surely one of these is redundant, but I worry which it is at this point? Please tell me both are not actually required for whatever accursed reason? (Also test failures look relevant to this bit directly)

Reply:
a crippling lazygit bug where it writes to two branches at once broke my pushes here. i'll fix..